index.bs

```diff
@@ -2448,8 +2448,7 @@ associated with or [=scoped=] to, respectively.
 
         When [=clients=], [=client platforms=], or [=authenticators=] display a {{PublicKeyCredentialEntity/name}}'s value, they should always use UI elements to provide a clear boundary around the displayed value, and not allow overflow into other elements [[css-overflow-3]].
 
-        [=Authenticators=] MUST accept and store a 64-byte minimum length for a {{PublicKeyCredentialEntity/name}} member's
-        value. Authenticators MAY truncate a {{PublicKeyCredentialEntity/name}} member's value so that it fits within 64 bytes. See [[#sctn-strings]] about truncation and other considerations.
+        Authenticators MAY truncate a {{PublicKeyCredentialEntity/name}} member's value so that it fits within 64 bytes. See [[#sctn-strings]] about truncation and other considerations.
```

Comment:
perhaps we ought to be explicit that authnrs MAY choose not to store the `{{PublicKeyCredentialEntity/name}} member's value` ?  I can go either way.

Reply:
There are other parameters that we are not forced to store and they are not explicitly called out. 
This was sort of an exception saying we had to store up to 64 bytes.   I don't think it needs more but can add something if people want.

Reply:
adding "if the authenticator stores the value" seems reasonable, but as you note, not required given precedent.

Reply:
Since previous versions of the spec required this to be stored, I think it's sensible to emphasise that we've changed our minds in the wording.

Reply:
ok, so is @ve7jtb going to add "if the authenticator stores the value" and then we can merge this?

Reply:
done